metacatalog/ext/standards_export/extension.py

```diff
@@ -78,6 +100,16 @@ def wrapper_api(session: Session, id_or_uuid: Union[int, str], config_dict: dict
         api.catalog.create_iso19115_xml = wrapper_api
 
 
+    @classmethod
+    def init_cli(cls, subparsers, defaults):
+        myparser = subparsers.add_parser('iso19115', parents=[defaults], help="Create ISO 19115 xml files")
```

Comment:
I am not sure if 'iso19115' is a good name though. 
You called the extension a standards extension, so we expect to add more standards in the future. Thus, I would already reflect that in the cli by setting an additional level of commands. in the end it should be called like:
`metacatalog export [entries] --format=iso19115

Reply:
or `metacatalog ext standards-export [entries] --format=iso19115`